From c9db0eeafd4659d9f38a6222aced8769d8963124 Mon Sep 17 00:00:00 2001
From: Wu Liang feng <wulf@rock-chips.com>
Date: Sat, 8 Oct 2016 11:26:40 +0800
Subject: [PATCH] arm64: dts: rockchip: set dwc3 dr_mode as otg for rk3399-box

rk3399-box Type-C0 USB needs to support peripheral mode
and host mode, so we set dr_mode as otg.

Change-Id: If94cdca3ec1d018c3f9aad14bb2c1e15e10e9c51
Signed-off-by: Wu Liang feng <wulf@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts | 2 +-
 arch/arm64/boot/dts/rockchip/rk3399-box.dtsi            | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
index ec6e7ab93522..2c35054ef6b2 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-808-android.dts
@@ -603,7 +603,7 @@
 };
 
 &usbdrd_dwc3_0 {
-	dr_mode = "peripheral";
+	dr_mode = "otg";
 	status = "okay";
 };
 
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index 910fde0a207e..11443aa59887 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -800,7 +800,7 @@
 };
 
 &usbdrd_dwc3_0 {
-	dr_mode = "peripheral";
+	dr_mode = "otg";
 	status = "okay";
 };
 
-- 
2.35.3

